src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GroupDto;
+import com.kodilla.ecommercee.mapper.GroupMapper;
+import com.kodilla.ecommercee.service.DbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/group")
+@CrossOrigin("*")
+public class GroupController {
+
+    @Autowired
+    private DbService service;
+
+    @Autowired
+    private GroupMapper groupMapper;
+
+    @RequestMapping (value = "v1/group/getGroupList", method = RequestMethod.GET)
```

Comment:
chyba to jest niezgodne z przyjętą konwencją:

GET /v1/product - wszystkie produkty
GET /v1/product/5 albo /v1/product?id=5 - product po ID
POST /v1/product - tworzenie nowego produktu
DELETE /v1/product/ - usuwanie produktu o id 5

Reply:
Czemu akurat /v1/product? skoro to nie jest ProductController.. może chodziło o /v1/Group itd.? 
Odnośnie konwencji, czy twoja uwaga tyczy się tylko konkretnie @RequestMappingu? Nie bardzo wiem, jak to inaczej napisać, bowiem w kontrolerze są dwa gety, z czego jeden już wykorzystałem tworząc @GetMapping, a kiedy w ten sam sposób próbowałem zrobić ten, o którym mowa, to nie chciało się skompilować. Nawet sam odpowiedziałeś jednemu z kolegów, że raczej nie może być dwóch @GetMappingów, bo wywala błąd, że już utworzono takiego beana. Stąd nie bardzo wiedziałem, jak to obejść inaczej, a w kursie jest @RequestMapping.

Reply:
To zapytaj na chacie jak ma byc

Reply:
Metoda get bez parametrów ma zwracać listę a z parametrem ma zwracać konkretny element. Dopytaj na chat